AUTHORS

```diff
@@ -1,21 +1,65 @@
-    Andy Polyakov
-    Ben Laurie
-    Bodo M�ller
-    Emilia K�sper
-    Eric Young
-    Geoff Thorpe
-    Holger Reif
-    Kurt Roeckx
-    Lutz J�nicke
-    Mark J. Cox
-    Matt Caswell
-    Nils Larsch
-    Paul C. Sutton
-    Ralf S. Engelschall
-    Rich Salz
-    Richard Levitte
-    Stephen Henson
-    Steve Marquess
-    Tim Hudson
-    Ulf M�ller
-    Viktor Dukhovni
+# This is the list of OpenSSL authors for copyright purposes.
+#
+# This does not necessarily list everyone who has contributed code, since in
+# some cases, their employer may be the copyright holder.  To see the full list
+# of contributors, see the revision history in source control.
+OpenSSL Software Services, Inc.
+OpenSSL Software Foundation, Inc.
+Akamai Technologies, Inc.
+Amadeus S.A.S.
+Amazon Fulfillment Services, Inc.
+Catena Cyber
+Chelsio Communications, Inc.
+Cisco Systems, Inc.
+Cloudflare, Inc.
+Cryptosoft Pty Ltd
+Disig A.S.
+Facebook, Inc.
+Google, Inc.
+IBM
+Intel Corporation
+L-3 Communications Linkabit
+LLC Samsung R&D Institute Russia
+Levitte Programming
+Miracl Limited
+NCP engineering GmbH
+NXP Semiconductors Netherlands BV
+Neutron Code Limited
+Nokia Solutions and Network Oy
+Oracle America Inc.
+Rambus, Inc.
+Red Hat, Inc.
+Ribose, Inc.
```

Comment:
I know another candidate that should be in here...

```
: ; git grep -n -C1 'Royal Institute'
CHANGES-5231-  *) Add processing of proxy certificates (see RFC 3820).  This work was
CHANGES:5232:     sponsored by KTH (The Royal Institute of Technology in Stockholm) and
CHANGES-5233-     EGEE (Enabling Grids for E-science in Europe).
--
crypto/x509v3/v3_pci.c-14- * Copyright (c) 2004 Kungliga Tekniska Högskolan
crypto/x509v3/v3_pci.c:15: * (Royal Institute of Technology, Stockholm, Sweden).
crypto/x509v3/v3_pci.c-16- * All rights reserved.
--
crypto/x509v3/v3_pcia.c-14- * Copyright (c) 2004 Kungliga Tekniska Högskolan
crypto/x509v3/v3_pcia.c:15: * (Royal Institute of Technology, Stockholm, Sweden).
crypto/x509v3/v3_pcia.c-16- * All rights reserved.
```

Reply:
The Swedish name is "Kungliga Tekniska Högskolan", translated to english as "Royal Institute of Technology, Stockholm, Sweden".  Choose the one you like, it doesn't matter as far as I know...